packages/vats/decentral-devnet-config.json

```diff
@@ -113,6 +113,15 @@
       "entrypoint": "defaultProposalBuilder",
       "args": [
         {
+          "contractTerms": {
+            "POLL_INTERVAL": 30,
+            "maxSubmissionCount": 1000,
+            "minSubmissionCount": 3,
+            "restartDelay": 1,
+            "timeout": 10,
+            "minSubmissionValue": 1,
+            "maxSubmissionValue": 1000000000000000000
```

Comment:
I'm not sure what's the impact of changing maxSubmissionValue from 2^256 to 1000000000000000000.

It would be great if the contract supported defaults.

Reply:
Perhaps use `9007199254740991`? aka
[Number.MAX_SAFE_INTEGER](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/MAX_SAFE_INTEGER)

Another option would be to change the contract to take maxSubmissionValueExponent and pass in 256.

Reply:
It doesn't seem like json will accept `Number.MAX_SAFE_INTEGER`. Did you mean to actually use `9007199254740991`?

Why would anyone set `maxSubmissionValue` to something other than `MAX_SAFE_INTEGER`?  In those circumstances are they going to want to distinguish 1 from 3 from 10^n?

Reply:
> Did you mean to actually use 9007199254740991?

Yes.

> Why would anyone set maxSubmissionValue to something other than MAX_SAFE_INTEGER?

I don't know the requirements around `maxSubmissionValue`; but `1000000000000000000` was just an arbitrary result of me mashing the keyboard while we were screensharing. The value on master is `2n ** 256n`. Of course it's not expressed in JSON:

https://github.com/Agoric/agoric-sdk/blob/720a16fd76a528868364bf107153c87e8a00e54c/packages/inter-protocol/scripts/price-feed-core.js#L11

Reply:
done

Reply:
It's still a lot smaller than 2^256. I can imagine it might matter in cases such as pricing DAI in USDC, where DAI uses 18 decimals and USDC uses 6. So this is a regression, and I'm not in a position to say whether it's OK.

@otoole-brendan is 9007199254740991 big enough?

I suppose if it's not, we can upgrade. So I guess I'll approve.